pigar/__main__.py

```diff
@@ -197,16 +203,20 @@ def extract_reqs(self):
         logger.info('Finish local environment checking.')
         return reqs, try_imports, guess
 
-    def _write_reqs(self, reqs):
+    def _write_reqs(self, reqs, minimal=False):
+        """Write requirements to a file"""
         print(Color.BLUE('Writing requirements to "{0}"'.format(
             self._save_path)))
         with open(self._save_path, 'w+') as f:
             f.write('# Requirements automatically generated by pigar.\n'
                     '# https://github.com/damnever/pigar\n')
-            for k, v in reqs.sorted_items():
+            if not minimal:
```

Comment:
Have you ever test it?

Reply:
@damnever I think I see. I changed the code so there is one blank line after the first comment, regardless of whether the option is used.

with --without-referenced-comments
```
# Requirements automatically generated by pigar.
# https://github.com/damnever/pigar

numpy == 1.18.1
opencv-contrib-python == 4.1.2.30
...
```

without the option
```
# Requirements automatically generated by pigar.
# https://github.com/damnever/pigar

# examples/Plant-Leaf-Disease-Detection/neural_network.py: 9
numpy == 1.18.1
...
```
Is that better?